core/assets/sounds/cues/spaceDanceJa/data.json

```diff
@@ -1,184 +1,204 @@
     },
     {
-      "id": "spaceDance/sit1",
+      "id": "spaceDanceJa/sit1",
+      "deprecatedIDs": ["spaceDance/sit1"],
       "duration": 0.5,
       "name": "dancers - let's sit down - sit"
     },
     {
-      "id": "spaceDance/down1",
+      "id": "spaceDanceJa/down1",
+      "deprecatedIDs": ["spaceDance/down1"],
       "duration": 0.5,
       "name": "dancers - let's sit down - down"
     },
     {
-      "id": "spaceDance/lets2",
+      "id": "spaceDanceJa/lets2",
+      "deprecatedIDs": ["spaceDance/lets2"],
       "duration": 0.5,
       "name": "gramps - let's sit down - let's"
     },
     {
-      "id": "spaceDance/sit2",
+      "id": "spaceDanceJa/sit2",
+      "deprecatedIDs": ["spaceDance/sit2"],
       "duration": 0.5,
       "name": "gramps - let's sit down - sit"
     },
     {
-      "id": "spaceDance/down2",
+      "id": "spaceDanceJa/down2",
+      "deprecatedIDs": ["spaceDance/down2"],
       "duration": 0.5,
       "name": "gramps - let's sit down - down"
     },
     {
-      "id": "spaceDance/pa1",
+      "id": "spaceDanceJa/pa1",
+      "deprecatedIDs": ["spaceDance/pa1"],
       "duration": 0.5,
       "name": "dancers - pu-pu-pu-punch - pu-"
     },
     {
-      "id": "spaceDance/punch1",
+      "id": "spaceDanceJa/punch1",
+      "deprecatedIDs": ["spaceDance/punch1"],
       "duration": 0.5,
       "name": "dancers - pu-pu-pu-punch - punch"
     },
     {
-      "id": "spaceDance/pa2",
+      "id": "spaceDanceJa/pa2",
+      "deprecatedIDs": ["spaceDance/pa2"],
       "duration": 0.5,
       "name": "gramps - pu-pu-pu-punch - pu-"
     },
     {
-      "id": "spaceDance/punch2",
+      "id": "spaceDanceJa/punch2",
+      "deprecatedIDs": ["spaceDance/punch2"],
       "duration": 0.5,
       "name": "gramps - pu-pu-pu-punch - punch"
     }
   ],
   "patterns": [
     {
-      "id": "spaceDance_turnright1",
+      "id": "spaceDanceJa_turnright1",
+      "deprecatedIDs": ["spaceDance_turnright1"],
       "name": "dancers - turn right",
       "cues": [
        {
-        "id": "spaceDance/turn1",
+        "id": "spaceDanceJa/turn1",
         "beat": 0.0
        },
        {
-        "id": "spaceDance/right1",
+        "id": "spaceDanceJa/right1",
         "beat": 1.0
        }
       ]
     },
     {
-      "id": "spaceDance_letssitdown1",
+      "id": "spaceDanceJa_letssitdown1",
+      "deprecatedIDs": ["spaceDance_letssitdown1"],
       "name": "dancers - let's sit down",
       "cues": [
        {
-        "id": "spaceDance/lets1",
+        "id": "spaceDanceJa/lets1",
         "beat": 0.0
        },
        {
-        "id": "spaceDance/sit1",
+        "id": "spaceDanceJa/sit1",
         "beat": 0.5
        },
        {
-        "id": "spaceDance/down1",
+        "id": "spaceDanceJa/down1",
         "beat": 1.0
        }
       ]
     },
     {
-      "id": "spaceDance_pupupupunch1",
+      "id": "spaceDanceJa_pupupupunch1",
+      "deprecatedIDs": ["spaceDance_pupupupunch1"]
```

Comment:
Missing comma. It's evident that you didn't test the PR with all sounds loaded because this error would've been caught.